app/models/validators/asylum_support_decision_validator.rb

```diff
@@ -14,6 +15,16 @@ class AsylumSupportDecisionValidator < SimpleDelegator
   validates :tribunal_decision_judges, presence: true
   validates :tribunal_decision_landmark, presence: true
   validates :tribunal_decision_reference_number, presence: true
-  validates :tribunal_decision_sub_category, presence: true
+  validates :tribunal_decision_sub_category, tribunal_decision_sub_category: true
```

Comment:
I can rename the validator class to `TribunalDecisionSubCategoryRelatesToParentValidator`. Then `tribunal_decision_sub_category_relates_to_parent: true` can be used in the call to validates().

As noted in the previous line comment discussion, sub-category validation is being extended based on a meeting with HMCTS. Some parent categories will not have any sub-categories related to them. A tribunal decision with a blank sub-category will be valid if the parent category has no sub-categories. Otherwise presence of the sub-category will be required. The commits for these changes will be added to the pull request soon.